features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/signicat/template.json

```diff
@@ -0,0 +1,101 @@
+            },
+            "userIdClaim": {
+                "uri": ""
+            }
+        },
+        "description": "",
+        "federatedAuthenticators": {
+            "defaultAuthenticatorId": "T3BlbklEQ29ubmVjdEF1dGhlbnRpY2F0b3I",
+            "authenticators": [
+                {
+                    "authenticatorId": "T3BlbklEQ29ubmVjdEF1dGhlbnRpY2F0b3I",
+                    "isEnabled": true,
+                    "properties": [
+                        {
+                            "key": "ClientId",
+                            "displayName": "Client ID",
+                            "description": "Enter client identifier value",
+                            "type": "STRING",
+                            "displayOrder": 1,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": false,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        },
+                        {
+                            "key": "ClientSecret",
+                            "displayName": "Client Secret",
+                            "description": "Enter client secret value",
+                            "type": "STRING",
+                            "displayOrder": 2,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": true,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        },
+                        {
+                            "key": "AuthenticationEndpointUrl",
+                            "displayName": "Authorization Endpoint URL",
+                            "description": "Authorization Endpoint URL value",
+                            "type": "STRING",
+                            "displayOrder": 1,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": false,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        },
+                        {
+                            "key": "TokenEndpointUrl",
+                            "displayName": "Token Endpoint URL",
+                            "description": "Token Endpoint URL value",
+                            "type": "STRING",
+                            "displayOrder": 1,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": false,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        }
```

Comment:
Since we are using OIDC authenticator we need to configure `OAuth2AuthzEPUrl`, `OAuth2TokenEPUrl`. I think these fields are incorrect. Also we have to add those values to the metadata.json file. Ref: https://github.com/wso2/carbon-identity-framework/blob/a9466fb03f491174d25749601a26e8e9428c0078/features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/microsoft/metadata.json#L114-L120

Reply:
Signicat's endpoint URLs could change based on the user's tenant and environment (sandbox or production). So we can't configure a static value like Microsoft